test/extension/commandExecutor.test.ts

```diff
@@ -87,7 +87,7 @@ suite("commandExecutor", function() {
 
             sinon.stub(Log, "log", function(message: string, formatMessage: boolean = true) {
                 loggedOutput += message;
```

Comment:
Seems to be there is no reason for using loggedOutput at all. I'd remove it.

Reply:
I agree.